gui/account.go

```diff
@@ -15,11 +15,11 @@ import (
 // account template.
 type accountPageData struct {
 	HeaderData       headerData
-	MinedWork        []minedWork
-	ArchivedPayments []archivedPayment
-	PendingPayments  []pendingPayment
+	MinedWork        *[]minedWork
```

Comment:
A slice is already a reference type, why are you taking the pointer there? Perhaps you're looking for `[]*minedWork` which is lightweight compared to `[]minedWork`.

Reply:
Hm good point. The idea was for the cache to just return a pointer to the slice it has created, because we dont need to create a copy, but as you say this is actually the default behaviour anyway. I will change it back